sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/implementation/AzureCommunicationCallAutomationServiceImplBuilder.java

```diff
@@ -191,7 +191,7 @@ public AzureCommunicationCallAutomationServiceImplBuilder addPolicy(HttpPipeline
      */
     public AzureCommunicationCallAutomationServiceImpl buildClient() {
         if (apiVersion == null) {
-            this.apiVersion = "2022-04-07-preview";
+            this.apiVersion = "2023-01-15-preview";
```

Comment:
I assume that 2023-01-15 works just the same as 2022-04-07, right?

Reply:
2023-01-15 would include play TTS, which has textSource model added.

Reply:
I meant, 2023 version is the same plus improvements.